src/operator/pairwise.ts

```diff
@@ -8,12 +8,13 @@ import {Subscriber} from '../Subscriber';
  * The (N-1)th is stored in the internal state until Nth input occurs.
  * @returns {Observable<R>} an observable of pairs of values.
  */
-export function pairwise<T>(): Observable<T> {
-  return this.lift(new PairwiseOperator());
+export function pairwise<T>(): Observable<[T, T]> {
+  let _this: Observable<T> = this;
```

Comment:
Got it. Thanks for the explanation.

I agree with you. This pattern leads to a better readability over casting. I'd prefer to name it `self` instead of `_this`, but this is a minor detail.